tests/integrated/test-drift-instability/CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 bout_add_integrated_test(test-drift-instability
   SOURCES 2fluid.cxx
+  REQUIRES NOT BOUT_USE_METRIC_3D
```

Comment:
This doesn't work because `REQUIRES` assumes this is two separate things. But just quoting them `"NOT BOUT_USE_METRIC_3D"` doesn't seem to work either. 

Ok, it turns out you can't just negate expressions. `NOT` really only works in `if()`. :(

Looks like this is fixable by adding a `CONFLICTS` argument to `bout_add_integrated_test`. I'll test that a bit and then push